eng/scripts/CodeCheck.ps1

```diff
@@ -113,13 +109,18 @@ try {
 
     Write-Host "Re-generating project lists"
     Invoke-Block {
-        & $PSScriptRoot\GenerateProjectList.ps1 -ci:$ci
+        & $PSScriptRoot\GenerateProjectList.ps1
+    }
+
+    Write-Host "Re-generating references assemblies"
+    Invoke-Block {
+        & $PSScriptRoot\GenerateReferenceAssemblies.ps1
     }
 
     Write-Host "Re-generating package baselines"
     $dotnet = 'dotnet'
     if ($ci) {
-        $dotnet = "$repoRoot/.dotnet/x64/dotnet.exe"
+        $dotnet = "$repoRoot/.dotnet/dotnet.exe"
```

Comment:
Undo. dotnet.exe is in a different location.

Reply:
Yep, already reverted CodeCheck changes.